src/Raven.Server/Documents/Replication/DocumentReplicationLoader.cs

```diff
@@ -270,6 +271,7 @@ private void InitializeResolvers()
 
         private void InitializeOutgoingReplications()
         {
+            _numberOfSiblings = 0;
```

Comment:
No, this is explicitly meant to be only set once, to avoid jitter on the value.
It is either the old value, or the new one, nothing in between.